Riot/Assets/en.lproj/Vector.strings

```diff
@@ -255,7 +255,11 @@
 "room_event_action_permalink" = "Permalink";
 "room_event_action_view_source" = "View Source";
 "room_event_action_report" = "Report content";
+"room_event_action_kick" = "Kick user";
+"room_event_action_ban" = "Ban user";
```

Comment:
`room_event_action_kick` and `room_event_action_ban` are not used.

Reply:
This is better to remove them.

Reply:
I removed those lines.